addons/calendar/controllers/main.py

```diff
@@ -66,6 +66,7 @@ def view_meeting(self, token, id, **kwargs):
         timezone = attendee.partner_id.tz
         lang = attendee.partner_id.lang or get_lang(request.env).code
         event = request.env['calendar.event'].with_context(tz=timezone, lang=lang).sudo().browse(int(id))
+        company = event.user_id.company_id
```

Comment:
It should be bullet proof when no user_id is defined on the event. That happens a lot.